README.md

```diff
@@ -43,9 +43,12 @@ Values: string
 - `@now-playing-paused-icon`  
 Description: A string to display for `{icon}` component when the music player
 is paused  
-Default: ` ` (empty space)
+Default: ` ` (empty space)  
+Values: string
+- `@now-playing-play-keytable`  
```

Comment:
It would be better to name it as `@now-playing-keytable` since it was not specific to only play action

Reply:
Thanks for your advice, it's ok.